tools/generators/lib/PBXProj/src/Optional+Extensions.swift

```diff
@@ -4,9 +4,10 @@ import ArgumentParser
 
 extension Optional: ExpressibleByArgument where Wrapped: ExpressibleByArgument {
     public init?(argument: String) {
-        guard !argument.isEmpty else {
-            return nil
+        if argument.isEmpty {
+            self = .none
```

Comment:
Any existing test already exercising this? If not worth adding one?

Reply:
No, but the future generators use, and they will error loudly if this fails. Now that it's fixed, not worth the effort imo.

Reply:
SGTM!